### PR TITLE
Minor Documentation Changes

### PR DESCRIPTION
**Pull request by Denton-L**

### CODE REVIEW

File: docs/solidity-by-example.rst [hunk 1]
```diff
@@ -377,29 +390,38 @@ high or low invalid bids.
...
+        )
             onlyAfter(biddingEnd)
             onlyBefore(revealEnd)
         {
             uint length = bids[msg.sender].length;
-            if (_values.length != length || _fake.length != length ||
-                        _secret.length != length)
+            if (
+                _values.length != length
+                || _fake.length != length
```
Comment (chriseth) [inline docs/solidity-by-example.rst]: I would prefer the `||` to go to the previous line (similar to what would happen for a `,`), but I'm not sure if that is explicit in our coding style.

File: docs/style-guide.rst [hunk 2]
```diff
@@ -157,15 +157,23 @@ Avoid extraneous whitespace in the following  situations:
 
 * Immediately inside parenthesis, brackets or braces.
 
-Yes: `spam(ham[1], Coin({name: "ham"}));`
+Yes::
```
Comment (chriseth) [inline docs/style-guide.rst:160]: Why have a bulleted list here at all?

**Comment by Denton-L** [comment]
Fixed, as specified.

**Comment by chriseth** [comment]
Thanks!

**Pull request merged**